explorer/app/components/Header/Header.tsx

```diff
@@ -1,9 +1,47 @@
 import React from "react";
+import { Logo } from "../Logo";
+import { NavLinks } from "../NavLinks";
+import { ProfileComponent } from "../ProfileComponent";
+import { Search } from "../Search";
+import { SocialLinks, SocialType } from "../SocialLinks";
 
 interface HeaderProps {
-  children: React.ReactNode | React.ReactNode[];
+  logo: {
```

Comment:
Would it be possible to create interfaces for Logo, NavLink and SocialLink and then use them here and also in the props of the corresponding components?

Reply:
Sure!